css/_vertical_filmstrip_overrides.scss

```diff
@@ -8,6 +8,8 @@
         display: flex;
         flex-direction: column-reverse;
         height: 100%;
+        position: fixed;
+        z-index: $tooltipsZ;
```

Comment:
Can we add this as a comment in the code. It will help people that aren't that familiar with css to understand the importance of this particular property here.